test/conformance/event/urEventWait.cpp

```diff
@@ -4,7 +4,7 @@
 #include <uur/fixtures.h>
 
 struct urEventWaitTest : uur::urQueueTest {
-    void SetUp() override {
+    void WriteSourceBuffer() {
```

Comment:
I don't think the negative tests should be creating buffers and copying memory when running. Negative tests should be as simple as possible. I couldn't think of a better way to handle this issue other than renaming SetUp and TearDown so that they have to be explicitly called. But I'm open to suggestions on how to handle this.

Reply:
Events are a bit different because they are implicitly tied to enqueuing commands to let the user determine when the start and finish and to create depedencies. OpenCL does have the concept of user events but we deliberately removed them from Unified Runtime because they are a pain to implement if its even possible when layering on top of e.g. CUDA.

Reply:
> Or do you think it's okay to setup buffers and copy memory on negative tests (even though those buffers are not actually used)?

I would have done this myself. Its a small amount of extra work but it could easiler just be a separate test fixture, no issue with that.

I'd much prefer keeping `SetUp()` and `TearDown()` rather than having these renamed member functions which are called manually.

Reply:
I ended up creating new fixtures. Because reusing the same fixtures would lead to memory leaks on negative tests because urEventRelease is not being called on TearDown.

Reply:
Sounds good 👍